### PR TITLE
✨ Ramp Credit KSM account

### LINKED ISSUE #2182: Revert back Credit component back
Toping up through Ramp network works again to comment of @cryptodamsky so let's return it back

Yet I don't have a commit on hand where we removed it recently

- https://github.com/kodadot/nft-gallery/blob/fd034e7229b69b9b22af8f0db7984815d35963d5/pages/rmrk/credit.vue

- https://docs.kodadot.xyz/how_to_top_up_wallet.html#top-up-through-ramp-network

### PR DESCRIPTION
**Pull request by roiLeo**
At the moment it's for KSM but it could be done depending on the choice of the indexer/prefix

### PR type

- [x] Feature

### What's new?

- [x] PR closes #2182 
- [x] fixed `/rmrk/credit` but page is hidden now
- [x] add RampNetwork SDK on click 'credit'

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Screenshot
Capture d’écran 2022-02-04 à 11 09 34 AM

### CODE REVIEW

File: components/common/RampCredit.vue [hunk 1]
```diff
@@ -43,6 +42,12 @@ const components = {
 
 @Component({ components })
 export default class RampCredit extends Vue {
```
Comment (vikiival) [inline components/common/RampCredit.vue:44]: please use
```ts
import AuthMixin from '@/utils/mixins/authMixin'
```

it's basically doing the same but it's just less code

File: components/common/RampCredit.vue [hunk 2]
```diff
@@ -10,18 +10,17 @@
...
         type="is-primary"
         tag="a"
         :href="
-          `https://ramp.network/buy/?swapAsset=KSM` +
+          `https://ramp.network/buy/?defaultAsset=KSM` +
           `&userAddress=${accountId}` +
           `&hostAppName=KodaDot` +
-          `&hostLogoUrl=https://nft.kodadot.xyz/kodadot_logo_v1_transparent_400px.png` +
+          `&hostLogoUrl=https://nft.kodadot.xyz/apple-touch-icon.png` +
           `&finalUrl=https://nft.kodadot.xyz` +
```
Comment (yangwao) [inline components/common/RampCredit.vue:22]:
```suggestion
          `&hostLogoUrl=https://kodadot.xyz/apple-touch-icon.png` +
          `&finalUrl=https://kodadot.xyz` +
```

Let's remove `nft.` prefix :)

File: components/rmrk/Profile/NavbarProfileDropdown.vue [hunk 3]
```diff
@@ -135,7 +136,18 @@ export default class NavbarProfileDropdown extends mixins(PrefixMixin) {
...
   }
 
-  checkExtension() {
+  protected showRampSDK(): void {
+    new RampInstantSDK({
+      defaultAsset: 'KSM', // todo: prefix
+      userAddress: this.account,
+      hostAppName: 'KodaDot',
+      hostApiKey: 'a99bfvomhhbvzy6thaycxbawz7d3pssuz2a8hsrc', // env
+      hostLogoUrl: 'https://nft.kodadot.xyz/apple-touch-icon.png',
```
Comment (yangwao) [inline components/rmrk/Profile/NavbarProfileDropdown.vue:145]:
```suggestion
      hostLogoUrl: 'https://kodadot.xyz/apple-touch-icon.png',
```
let's remove `nft.` prefix

**Review by yangwao — CHANGES_REQUESTED** [review]
let's remove `nft.`

**Comment by yangwao** [comment]
haven't tested but looks good 

image

**Pull request merged**